### ISSUE TITLE
AV1 Temporal Delimiters misplaced

### ISSUE THREAD

**Issue by cconcolato** [gpac/gpac#2681]
The following command line is not working:
```
gpac -i source.obu rfav1:temporal_delim -o td.mp4
```
The Temporal Delimiter of the first Temporal Unit is dropped and for consecutive TUs, the TD of the next TU is added at the end of the previous MP4 sample, so the last sample does not have any at all, as reported by:
```
gpac -i td.mp4 inspect:deep:analyze=on:log=log.xml
```

**Comment by rbouqueau** [comment]
Indeed, thanks for reporting. Should be fixed, can you check and close?

**Comment by cconcolato** [comment]
It looks good to me in the normal case. In the edge case of a stream made of 1 frame, it does not seem to work
```
gpac -i out_track1_001.obu rfav1:temporal_delim -o 1.mp4
Filter ffdmx failed to setup: Filter not found for the desired type
session last connect error Filter not found for the desired type
Filters not connected:
rfav1 (temporal_delim) (idx=1)
fout (dst=1.mp4) (idx=2)
```
`out_track1_001.obu` is the result of `MP4Box -raws 1 out.mp4` when `out.mp4` was produced using gpac as above

**Comment by rbouqueau** [comment]
I don't reproduce your specific issue. Would you mind sharing your sample?

The  ```temporal_delim``` option just keeps the TDs, it doesn't create new ones.

What I get:

```
$ gpac -i av1.obu rfav1:temporal_delim -o td.mp4
$ MP4Box -raws 1 td.mp4
$ gpac -i td_track1_001.obu  rfav1:temporal_delim -o 1.mp4
```

and 

```
$ MP4Box -raws 2 av1.mp4
$ gpac -i av1_track2_1.obu  rfav1:temporal_delim -o 1.mp4
[AV1Dmx] Error OBU stream start with frame, not a temporal delimiter - NOT SUPPORTED
Filter rfav1 failed to setup: No Error

Filters not connected:
fout (dst=1.mp4) (idx=1)
```

**Comment by rbouqueau** [comment]
@cconcolato (in case you missed the notification)

**Comment by rbouqueau** [comment]
Can we close now?

**Comment by jeanlf** [comment]
should be fixed by 22e7bd7e
